tests/vmi_configuration_test.go

```diff
@@ -2963,7 +2963,7 @@ var _ = Describe("[sig-compute]Configurations", func() {
 		})
 	})
 	Context("virt-launcher processes memory usage", func() {
-		It("should be lower than allocated size", func() {
+		FIt("should be lower than allocated size", func() {
```

Comment:
This line has a focus. This will break the test suite.

Reply:
This will only break local development.

Reply:
Thanks both <3